articles/Monerium/monerium.md

```diff
@@ -0,0 +1,155 @@
+
+In a similar manner, EURe can be off-boarded. For that, a user needs to hold EURe on the on-chain address which corresponds to his IBAN. He then initiates the withdrawal process to a bank account, whereupon an equivalent amount is automatically burned from the on-chain wallet.
+
+Currently, the process of burning/withdrawing the stablecoins is being approved by hand. In the future, this process will be scaled to allow automatic withdrawals up to a certain threshold that will also be cleared within seconds through SEPA Instant.
+
+Furthermore, Monerium will add the option to change the on-chain address to enhance the privacy of its customers.
+
+### Regulatory & Legal
+
+Monerium is an authorised electronic money institution, granted authorization to issue electronic money according to Act no. 17/2013, on issuance and handling of electronic money, Icelandic Act implementing Directive 2009/110/EC. Monerium provides its services online but its registered office is at Bjargargata 1, 102 Reykjavík, Iceland (Company number 5711100240). The company is supervised by the Financial Supervisory Authority of the Central bank of Iceland. ([See here for further information](https://monerium.com/policies/business-terms-of-service/))
+
+Additional [regulatory requirements](https://eur-lex.europa.eu/legal-content/EN/TXT/?uri=CELEX%3A52020PC0593) ([Markets in Crypto Assets Regulation - MiCA](https://www.sidley.com/en/insights/newsupdates/2022/11/how-will-the-eu-markets-in-crypto-assets-regulation-affect-crypto-and-other-financial-services-firms)) are currently in the works which would impose further obligations on fiat-referenced tokens (e-money tokens). According to MiCA, such tokens should not only adhere to the EMD but also adhere to requirements such as [diligent marketing](https://eur-lex.europa.eu/legal-content/EN/TXT/?uri=CELEX%3A52020PC0593#:~:text=Article%2048%0A%C2%A0%C2%A0%C2%A0Marketing%20communications), [disclosure of risks](https://eur-lex.europa.eu/legal-content/EN/TXT/?uri=CELEX%3A52020PC0593#:~:text=the%20risks%20relating%20to%20the%20issuer%20of%20e%2Dmoney%20issuer%2C%20the%20e%2Dmoney%20tokens%20and%20the%20implementation%20of%20the%20project%2C%20including%20the%20technology%3B), [prohibition of interest](https://eur-lex.europa.eu/legal-content/EN/TXT/?uri=CELEX%3A52020PC0593#:~:text=Prohibition%20of%20interests), and more. Currently, MiCA has not entered into force however it is expected that this will happen [somewhere around Q1 2023, after which it will come into effect in 2024](https://www.iflr.com/article/2a647jipe3beilngwqlmo/primer-markets-in-crypto-assets-regulation-mica#:~:text=The%20European%20Council%20published%20the,the%20requirements%20of%20the%20regulation.) when implemented by the member states of the EU. In this regard, Monerium is already coherent with future regulations by issuing their euro stablecoin under the EMD.
+
```

Comment:
You listed three different options in the previous sections wrt backing, what option did Monerium go for? Did they take additional precautions? (For instance, taking an insurance policy on top of having them invested in low risk assets)

Reply:
Covered in 
### Risk Summary e-money:

> Secondly, most of the assets are invested into high-quality liquid assets and only the necessary amount is kept at a bank to meet regular withdrawals (as part of a prudent practice), whereby the reserves are also segregated from the EMI.

Reply:
Ok, I wasn't sure if this you were talking about Monerium specifically here or making a general statement about what EMIs generally do, since the subsection is called "Risk Summary E-money" and talks about general features of EMI/e-money before getting into the specifics into the dedicated "Monerium" section next.

Maybe rephrase this sentence to say this is specifically what Monerium is doing (could also mention that they're not insured) or move/repeat it in the Monerium section (with all the EU legalese in the first section wouldn't be surprised if a lot of readers skip :sweat_smile:

Reply:
All customer funds are safeguarded in segregated customer accounts and they are not commingled with the company's accounts. The law protects against insolvency of the EMI in such a way that holders of EURe have a direct claim on the safeguarded funds which has a priority above everything else, including employee salary payments. The cool thing is that because the EURe is minted on a blockchain, proving the claim is trivial.